Makefile

```diff
@@ -85,6 +107,12 @@ help:
 	@echo "    help:               Print this usage information."
 	@echo "    man:                Generate all man-pages"
 	@echo
+	@echo "    rpm:                Generate the rpm to install osbuild"
+	@echo "    srpm:               Generate the source rpms"
+	@echo "    rpm-devel:          Generate development rpms without checking consistency"
```

Comment:
Normally the rpm/srpm targets check if there are any uncommitted changes. It doesn't do that here (the comment could be better).

Reply:
The `rpm` target does more than just wrapping it into an rpm (checking `git-diff-check`) but I'm tempted to remove building an rpm just to get the source running in a container (which is the target of the `getting started` repo)

Reply:
Thanks! Maybe `Generate development rpms that include uncommited changes` then? Fwiw, jus doing a "setup.py install" (or something like this) in the Devcontainer seems fine to me, I have no strong opinion (might be nicer as it would remove the concern about mixing a "real" release and a "dev" release that requires (AIUI) the two different rpm targets.

Reply:
This conversation is marked "resolve" but I am not sure why? The "echo" did not change or a comment about it made, am I missing something?

Reply:
I thought the comment was explained and ok-ish. I took over your suggestion now, which is better.